wsgi/odyssy/basic/templates/basic/index.html

```diff
@@ -36,17 +36,17 @@
             <!-- Announcements -->
             <div class="col-sm-4" id="announcements">
                 <div class='announcement-content '>
-                    <div class="widgets-title"><h2>Announcements</h2></div>
+                    <div class="widgets-title"><h2><a href="/announcement/">Announcements</a></h2></div>
```

Comment:
@mohitkyadav yes there is a view which you are talking about but that html file is named as `index.html` in announcement app and its name as `index-view` unlike other apps, where they are named as `event_list.html` or `news_list.html` and have a name similar to this.